modules/issue_tracker/jsx/attachments/attachmentsList.js

```diff
@@ -100,7 +100,7 @@ class AttachmentsList extends Component {
    */
   displayAttachmentOptions(deleteData, item) {
     if (this.props.userHasPermission
-      || this.state.attachments.whoami === item.user) {
+      || this.state.issueData.whoami === item.user) {
```

Comment:
issueData isn't a defined state in this class. not sure what this is supposed to be?